docs/source/faq.rst

```diff
@@ -714,3 +714,66 @@ Optuna may sometimes suggest parameters evaluated in the past and if you would l
 
     study = optuna.create_study()
     study.optimize(objective, n_trials=100)
+
+How can I sample the best expected parameters based on the evaluated trials?
```

Comment:
I think some people need the parameters without sampling a new trial, so maybe it's beneficial to introduce the non-sampler way that directly uses `_gp` classes?

Reply:
Also it should be pointed out that this is a "hack" and it is not officially supported. It is possible that the interface changes in the future.